client/verta/verta/_registry/modelversion.py

```diff
@@ -691,7 +693,8 @@ def download_docker_context(self, download_to_path):
             self._conn.socket,
         )
         body = {
-            "model_version_id": self.id
+            "model_version_id": self.id,
+            "self_contained": self_contained
```

Comment:
Personal preference: trailing commas makes future diffs easier on the eyes
```suggestion
            "self_contained": self_contained,
```

Reply:
Fixed.